libr/bin/pdb/pdb_downloader.c

```diff
@@ -23,15 +23,28 @@ static bool download_and_write(SPDBDownloaderOpt *opt, const char *file) {
 		return false;
 	}
 	char *url = r_str_newf ("%s/%s/%s/%s", opt->symbol_server, opt->dbg_file, opt->guid, file);
+	char *path = r_str_newf ("%s%s%s", dir, R_SYS_DIR, opt->dbg_file);
+#if __WINDOWS__
+	if (r_str_startswith (url, "\\\\")) { // Network path
```

Comment:
so, this is in case you are on windows and the pdb file is somewhere on the network of the host? In that case, windows is capable of directly getting the file with a CopyFileW instead of using httpget? Did I get it right?

Reply:
Yes indeed, you can check all the valid file names [here](https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file) that CopyFileW can take.